src/module/nft/nft.rs

```diff
@@ -0,0 +1,180 @@
+use std::{
+    mem,
+    process::{Command, Stdio},
+};
+
+use anyhow::{anyhow, bail, Result};
+use clap::{arg, builder::PossibleValuesParser, Parser};
+use serde_json::json;
+
+use super::{bpf::*, nft_hook};
+use crate::{
+    cli::{dynamic::DynamicCommand, CliConfig},
+    collect::{cli::Collect, Collector},
+    core::{
+        kernel::Symbol,
+        probe::{Hook, Probe, ProbeManager},
+    },
+    module::ModuleId,
+};
+
+static NFT_BIN: &str = "nft";
+
+#[derive(Parser, Default)]
+pub(crate) struct NftCollectorArgs {
+    #[arg(
+        long,
+        value_parser=PossibleValuesParser::new(["ALL", "CONTINUE", "BREAK", "JUMP", "GOTO", "RETURN", "DROP", "ACCEPT", "QUEUE", "REPEAT"]),
+        value_delimiter=',',
+        default_value="DROP,ACCEPT",
+        help = "Comma separated list of verdicts whose events will be collected"
+    )]
+    nft_verdicts: Vec<String>,
+}
+
+#[derive(Default)]
+pub(crate) struct NftCollector {
+    side_effects: bool,
+}
+
+impl NftCollector {
+    fn probe_bin(&self) -> Result<()> {
+        Command::new(NFT_BIN)
+            .arg("-v")
+            .spawn()
+            .map_err(|e| anyhow!("{NFT_BIN} binary not available - {e}"))?;
```

Comment:
I used the hyphen as a separator to avoid multiple colon.

`Error: Could not initialize the Nft collector: nft binary not available: No such file or directory (os error 2)`

Let's move back to colon